### PR TITLE
Add basic prompts: `airplane.input.text` 

### PR DESCRIPTION
**Pull request by colinking**
Adds support for a single type of prompt: text strings. Follow-up PRs will add the remainder of the prompt kinds.

cc @kellymtrinh 

Resolves AIR-4114

### CODE REVIEW

File: internal/input/slug.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const maxLength = 50;
+
+// makeSlug will convert a string into a valid slug, according to the
+// semantics of slugs used in the Airplane API. Emulates the
+// same gosimple/slug-based implementation.
+export const makeSlug = (s: string): string => {
```
Comment (colinking) [inline internal/input/slug.ts:6]: Ported over from `airplanedev/web`. Would be nice to share as a package at some point, but just duplicating for now.

File: internal/api/client.ts [hunk 2]
```diff
@@ -47,20 +47,34 @@ export class Client {
...
-      paramValues: ParamValues;
+      paramValues: P;
       taskID: string;
     }>("/v0/runs/get", { id: runID });
   }
+
+  async createPrompt(params: ParamSchema[]): Promise<string> {
+    const resp = await this.fetcher.post<{
+      id: string;
+    }>("/v0/prompts/create", { schema: { parameters: params }, values: null });
```
Comment (francescaleung) [inline internal/api/client.ts:70]: is it weird that this needs to send values?

**Review by francescaleung — APPROVED** [review]

File: internal/api/client.ts (hunk 2 above)
Comment (colinking) [reply internal/api/client.ts:70]: Ah, it doesn't need to. The idea is you can submit "default values". We don't use that feature yet, though. Will remove from the client here.

**Comment by colinking** [comment]
@francescaleung I'm also bumping the version to `-12`

**Pull request merged**